src/Shell/Task/ControllerTask.php

```diff
@@ -46,7 +46,7 @@ class ControllerTask extends BakeTask
      * Execution method always used for tasks
      *
      * @param string|null $name The name of the controller to bake.
-     * @return void
+     * @return void|bool
```

Comment:
null|bool
void by definition cannot be mixed with others
either you are supposed to check on the return value (non-void) or you are not (void).
mixing void with non-void means this behavior is undefined ;)